redis/src/cluster_async/connections_logic.rs

```diff
@@ -429,10 +437,36 @@ async fn create_connection<C>(
 where
     C: ConnectionLike + Connect + Send + 'static,
 {
+    let retry_strategy = ExponentialBackoff {
+        initial_interval: std::time::Duration::from_millis(
+            params.exponential_backoff.factor as u64,
+        ),
+        multiplier: params.exponential_backoff.exponent_base as f64,
+        max_elapsed_time: None,
```

Comment:
We should have max elapsed time set to tens of minutes (it this is that parameter means)